core/pom.xml

```diff
@@ -340,20 +340,6 @@ THE SOFTWARE.
     <dependency>
       <groupId>org.jvnet.hudson</groupId>
       <artifactId>commons-jelly-tags-define</artifactId>
-      <exclusions>
-        <exclusion>
-          <groupId>commons-cli</groupId>
-          <artifactId>commons-cli</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>dom4j</groupId>
-          <artifactId>dom4j</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>org.jvnet.hudson</groupId>
-          <artifactId>commons-jelly</artifactId>
-        </exclusion>
```

Comment:
Under certain circumstances involving unusual parent POMs or BOMs, a plugin with a 2.389+ dep may encounter a test failure

```
java.lang.NoSuchMethodError: 'void org.kohsuke.stapler.jelly.CustomJellyContext$CustomXMLParser.setEscapeByDefault(boolean)'
	at org.kohsuke.stapler.jelly.CustomJellyContext$CustomXMLParser.<init>(CustomJellyContext.java:131)
	at org.kohsuke.stapler.jelly.CustomJellyContext.createXMLParser(CustomJellyContext.java:97)
	at org.apache.commons.jelly.JellyContext.getXMLParser(JellyContext.java:556)
	at org.apache.commons.jelly.JellyContext.compileScript(JellyContext.java:516)
	at org.jvnet.hudson.test.JellyTestSuiteBuilder$JellyCheck.runTest(JellyTestSuiteBuilder.java:102)
```

One fix is to add to the POM

```xml
<dependencyManagement>
  <dependencies>
    <dependency>
      <groupId>org.jenkins-ci.main</groupId>
      <artifactId>jenkins-bom</artifactId>
      <version>${jenkins-bom.version}</version>
      <type>pom</type>
      <scope>import</scope>
    </dependency>
  </dependencies>
</dependencyManagement>
```

reinforcing what `plugin-pom` normally does.